Orange/canvas/__main__.py

```diff
@@ -271,6 +272,16 @@ def compare_versions(latest):
         thread.start()
         return thread
 
+def send_usage_statistics():
+
+    class SendUsageStatistics(QThread):
```

Comment:
What happens if there is an uncaught exception in a QThread for any reason? I faintly remember some problems with crashing and uncaught exceptions in threads, but that might be just my imagination.

Also, as this addition is a non-vital part of Orange, I recommend being defensive. See send_usage_statistics.

Reply:
Did you mean like this? I added an except Exception to send_statistics.

Reply:
I added a commit that moves checking for exception to run(). Because, for example, an exception could also happen in json loading (if file was saved incorrectly). Could you verify if everything is still OK?